docs/platform/atoms/booker.mdx

```diff
@@ -24,6 +24,66 @@ export default function Booker( props : BookerProps ) {
 }
 ```
 
+## Dynamic Bookings
+
+Dynamic bookings allow you to show combined availability across multiple users. This is useful for scenarios like:
+- Group meetings where all participants must be available
+- Sales calls that require multiple team members
+- Consultations with multiple experts
+
+### Using Dynamic Bookings
+
+Pass an array of usernames to create a dynamic booking:
+
+```js
+import { Booker } from "@calcom/atoms";
+
+export default function DynamicBooker() {
+  return (
+    <Booker
+      // Pass multiple usernames for combined availability
+      username={["alice", "bob", "charlie"]}
+      eventSlug="group-meeting"
+      onCreateBookingSuccess={() => {
+        console.log("Dynamic booking created!");
+      }}
+    />
+  )
+}
+```
+
+<Note>
+**Requirements for Dynamic Bookings:**
+- All specified users must have `allowDynamicBooking` enabled in their settings
+- The event type slug should be a valid dynamic event type (or use "dynamic" as the slug)
+- For organization-scoped dynamic bookings, provide the `entity.orgSlug` prop
+</Note>
+
+### Dynamic Booking with Organization Context
+
+When booking within an organization, provide the `entity` prop to ensure correct availability resolution:
```

Comment:
Since you added function `resolveOrgSlug` which resolves it from the `event.data` we don't need to pass entity anymore.